sql/plan/scope.go

```diff
@@ -12,11 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package analyzer
+package plan
```

Comment:
Probably a better match for the sql package than the plan package

Reply:
Maybe, my thought process was that scope is probably going to be mostly limited to stored procedures soon and will have the lifespan of the `plan` package.